examples/pipeline/README.md

```diff
@@ -38,10 +38,12 @@ Overview
   make tf-apply
   ```
 6. Disable the creations of All AWS Accounts and Custom SCPs in your TF Vars file
-  1. Comment out the `accounts = {}` block
-  2. Comment out the `service_control_policies = {}` block
-  3. We will re-enable them after the first apply.
-  4. Delete the `security_services.tf` file on the first run.
+  1. **IMPORTANT**: Ensure that your `payer_name` and `payer_email` variables align with what you manually used when creating the first AWS account, otherwise
```

Comment:
I feel like the importance of this one can't be understated; perhaps break it out to be even _more_ obvious?